.github/workflows/python.yml

```diff
@@ -27,22 +27,26 @@ jobs:
       - name: "Install Rust 1.85.0"
         uses: dtolnay/rust-toolchain@1.85.0
 
-      - name: "Install Python"
-        uses: actions/setup-python@v4
-        with:
-          python-version: 3.12
-
       - name: "Use cache"
         uses: Swatinem/rust-cache@v2
 
+      - name: Install a specific version of uv
+        uses: astral-sh/setup-uv@v6
+        with:
+          version: "0.8.2"
```

Comment:
This is simply the current version of `uv`, I figured we shouldn't just arbitrarily pull latest?